user-config-example.yaml

```diff
@@ -0,0 +1,187 @@
+rootUrl: ...
+dockerImage: ...
+ingressStaticIpName: ...
+ingressCertName: ...
+auth:
+  client_table_name: ...
+  roles_container_name: ...
+  static_clients: {}
+  azure_accounts: {}
+  sentry_organization: ...
+  sentry_host: ...
+  sentry_auth_token: ...
+  sentry_team: ...
+  sentry_key_prefix: ...
+  statsum_api_secret: ...
+  statsum_base_url: ...
+  websocktunnel_secret: ...
+  docs_bucket: ...
+  schemas_bucket: ...
+  refs_bucket: ...
+  audit_log: ...
+  azure_account: ...
+  azure_account_key: ...
+  azure_signing_key: ...
+  azure_crypto_key: ...
+  pulse_username: ...
+  pulse_password: ...
+  pulse_hostname: ...
+  pulse_vhost: ...
+  aws_access_key_id: ...
+  aws_secret_access_key: ...
+  aws_region: ...
+  gcp_credentials: {}
+  test_bucket: ...
+built_in_workers:
+  taskcluster_client_id: ...
```

Comment:
There's no reason for people to customize these from the standard names like  "static/taskcluster/github" , is there?

I think all the `*_client_id` could go in variables.yaml.